TypeScript.YAML-tmLanguage

```diff
@@ -402,11 +405,11 @@ repository:
 
   relational-operator:
     name: keyword.operator.comparison.ts
-    match: ===|==|=|!=|!==|<=|>=|<>|<|>
+    match: ===|==|!=|!==|<=|>=|<>|<|>
```

Comment:
Bump on this... did this ever get corrected? (Or a test added and me proved wrong? :smile: )